presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -4,6 +4,7 @@ Release 0.245
 
 .. warning::
     There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
+    There is a bug causing failure when reading ORC files having ARRAY/MAP/ROW of VARCHAR columns for some corner cases, and the fix is (:pr:`15549`).
```

Comment:
I would also mention `when using the SelectiveStreamReaders`

Reply:
@bhhari Thanks for reviewing! I think the users might not know the code `SelectiveStreamReaders`. How about saying "when Aria scan is enabled"?

Reply:
This is my suggestion (incorporating Ying's suggestion for Aria scan):

```
There is a bug causing failure when reading ORC files having ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns for some corner cases when using Aria scan. (:pr:`15549`)
``

Reply:
Updated to be
```
Fix a bug reading ORC files having ARRAY/MAP/ROW of VARCHAR columns using selective stream readers for some corner cases
```

Reply:
@mayankgarg1990 Thank you fore reviewing! @bhhari thinks users may not know what Aria is, so I changed this to
```
There is a bug causing failure when reading ORC files with ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns using the selective stream readers for some corner cases 
```
How do you like it?

Reply:
sounds reasonable